build/scripts/sanitycheck.py

```diff
@@ -8,12 +8,21 @@
 CRLF = b'\r\n'
 LF = b'\n'
 
+# Add paths to exclude from sanity checks here
+exclude_folders = [
+    "src/OpenTelemetry.SemanticConventions"
```

Comment:
Can it be more granular? Only for generated subfolders?

Reply:
Makes sense, done!